src/fmain.pas

```diff
@@ -1039,22 +1042,6 @@ procedure TfrmMain.FormCreate(Sender: TObject);
     Result.OnDragOver:= @NotebookDragOver;
     Result.OnDragDrop:= @NotebookDragDrop;
   end;
-  function GenerateTitle():String;
```

Comment:
In this case no need to move function to another place.

Reply:
In case `GenerateTitle()` declaration and implementations was placed in original position, I could not use it in `UpdateMainTitleBar`: "Error: (5000) Identifier not found "GenerateTitle"". Moving declaration + implementation near UpdateMainTitleBar solved that...

Reply:
But you don't use it in "UpdateMainTitleBar", variable "sStaticTitleBarString" used here instead. "GenerateTitle()" is used only in "FormCreate" as is before.

Reply:
Ok :)